manifests/base/image.yaml

```diff
@@ -3,7 +3,7 @@ kind: Qserv
 metadata:
   name: qserv
 spec:
-  image: qserv/lite-qserv:2022.1.1-rc1
+  image: qserv/lite-qserv:2022.1.1-rc1-50-gd066e4e64
```

Comment:
May wish to update these to `2022.2.1-rc1` at this point?

Reply:
I'll upgrade to `2022.2.1-rc1` in a new ticket.